cmd/pcap-broker/main.go

```diff
@@ -115,7 +115,9 @@ func main() {
 	log.Debug().Strs("args", args).Send()
 
 	cmd.Stdout = wStdout
-	cmd.Stderr = log.Logger
+	cmd.Stderr = log.Logger.Hook(zerolog.HookFunc(func(e *zerolog.Event, level zerolog.Level, msg string) {
+		e.Str(zerolog.LevelFieldName, zerolog.LevelDebugValue)
```

Comment:
If we use the TRACE level we can make the messages a bit more distinct from the rest of the DEBUG messages. 

```suggestion
		e.Str(zerolog.LevelFieldName, zerolog.LevelTraceValue)
```